modules/apps/frontend-data-set/frontend-data-set-web/src/main/resources/META-INF/resources/management_bar/components/filters/implementation/dateRangeFilter.tsx

```diff
@@ -13,9 +13,41 @@ import {
 	formatDateObject,
 	formatDateRangeObject,
 	getDateFromDateString,
-} from '../../../utils/dates';
 
-const getIsoString = ({direction, entityFieldType, objectDate}) => {
+	// @ts-ignore
```

Comment:
Should the file name be `DateRangeFilterImplementation.tsx`? lowercase first latter is what bugs me the most, I would expect this file to export a function with the same name.

Reply:
For the same reason I would expect a file named `DateRangeFilterImplementation.tsx` to return a class :sweat_smile:, not a constructor and two functions :sweat_smile: .

But if our convention for naming files that return mixed things is that one, I'm OK. TBH, I don't care too much about naming as long as it is consistent throughout the code...

Reply:
Looking at [guidelines](https://github.com/liferay/liferay-frontend-projects/blob/master/guidelines/general/file_names.md), there is no case where we export a single default object :man_shrugging:  I'd say the closest content type to what we have here is single default class, implying PascalCase.

Reply:
done